watertap/tools/oli/mcas_flowsheet.py

```diff
@@ -0,0 +1,123 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+#################################################################################
+
+__author__ = "Paul Vecchiarelli, Adam Atia"
+
+from pyomo.environ import ConcreteModel, assert_optimal_termination
+from idaes.core import FlowsheetBlock
+from idaes.core.util.scaling import calculate_scaling_factors
+from idaes.core.solvers import get_solver
+import watertap.property_models.multicomp_aq_sol_prop_pack as props
+
+from watertap.tools.oli.watertap_to_oli_helper_functions import watertap_to_oli
+
+solver = get_solver()
+
+
+def mcas_flowsheet(source_water: dict):
+    """
+    Create a simple MCAS Flowsheet from a feedwater dictionary.
```

Comment:
```suggestion
    Create a simple MCAS StateBlock from a feedwater dictionary.
```
Even though we technically build a FlowsheetBlock (as we do for any of our models), I think we should refer to this as creating an MCAS stateblock instead of an MCAS flowsheet.